ui/src/lib.ts

```diff
@@ -12,33 +13,37 @@ const formatDate: Function = (date: Date): string => {
   ].join("-");
 };
 
-const formatDateMinusOneDay: Function = (date: Date): string => {
+const dateMinusOneDay: Function = (date: Date): Date => {
   // subtract a day from the given date
   date.setDate(date.getDate() - 1);
-  return [
-    date.getFullYear(),
-    padNumberWith0Zero(date.getMonth() + 1),
-    padNumberWith0Zero(date.getDate()),
-  ].join("-");
+  return date;
 };
 
-const dateFormat = (selectedEventDate: Date) =>
-  `${selectedEventDate.toLocaleString("default", {
+const dateFormat = (date: Date) =>
+  `${date.toLocaleString("default", {
     weekday: "short",
-  })}, ${selectedEventDate.toLocaleString("default", {
+  })}, ${date.toLocaleString("default", {
     month: "short",
-  })} ${selectedEventDate.getDate()}`;
+  })} ${date.getDate()}`;
 
-const dateFormatWithYear = (selectedEventDate: Date) => {
-  return `${selectedEventDate.toLocaleString("default", {
+const dateFormatWithYear = (date: Date) => {
+  return `${date.toLocaleString("default", {
     weekday: "short",
-  })}, ${selectedEventDate.toLocaleString("default", {
+  })}, ${date.toLocaleString("default", {
     month: "short",
-  })} ${selectedEventDate.getDate()} ${selectedEventDate.getFullYear()}`;
+  })} ${date.getDate()} ${date.getFullYear()}`;
 };
 
-const timeFormat = (selectedEventDate: Date) =>
-  `${selectedEventDate.toLocaleTimeString("default", {
+// ex. 13:00
```

Comment:
Though there is something nice about seeing an actual result here maybe we could put JSDocs to specify the return results:
![Screen Shot 2023-01-09 at 9 46 48 PM](https://user-images.githubusercontent.com/50315144/211471340-5da19293-9aed-4e7a-bbc4-27a8545c2860.png)

This seems like a potential new convention adding comment/JSDoc with an example of expect result that may be nice to go over as a team so we're all on the same page or get feedback on the convention before continued use.

Reply:
Though some parts of JSDoc are redundant when we already define our types with Typescript so maybe just the @returns section could be useful or having it in comment format as you do. Let's check with the team

Reply:
Here's a nice article I found on the usage of JSDoc: https://www.prisma.io/blog/type-safe-js-with-jsdoc-typeSaf3js#:~:text=JsDoc%20provides%20a%20way%20to,undeclared%20variables%2C%20and%20undefined%20types.

Reply:
Updated here to include a description 8e55de2